plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/embeddings/FieldEmbedding.kt

```diff
@@ -44,6 +43,11 @@ interface FieldEmbedding {
             AccessPolicy.ALWAYS_INHALE_EXHALE -> FieldAccessTypeInvariantEmbedding(this, PermExp.FullPerm())
             AccessPolicy.ALWAYS_READABLE, AccessPolicy.ALWAYS_WRITEABLE -> null
         }
+
+    val typesContainingInPrimaryConstructor: Map<TypeEmbedding, FirPropertySymbol>
```

Comment:
I'm confused by this name.  Do you mean the types that are present in the primary constructor?  I would use `present` instead of `containing` then.  If not, what do they contain?